app/models/plaid_item.rb

```diff
@@ -1,6 +1,9 @@
 class PlaidItem < ApplicationRecord
   include Plaidable, Syncable
 
+  enum :plaid_region, { us: "us", eu: "eu" }
+  validates :plaid_region, inclusion: { in: plaid_regions.keys }
```

Comment:
The `enum` should have this `inclusion` validation already built-in, so we can remove the second `validates` here